### DISCUSSION TITLE
Connectivity failure in DAX Studio

### DISCUSSION THREAD

**Discussion by SenthilkumarE** [DaxStudio/DaxStudio#711] [Q&A]
I am trying to connect my Azure SSAS Tabular but i am getting "Error Connecting : Cannot connect to Server "ServerName" when its paused or scaling. Please Help.

**Comment by dgosbell** [comment]
What version of DAX Studio are you using? Is it the latest (v2.16.2)? As Microsoft are continually updating the client libraries and it's possible that you could see unexpected errors if you are using an older version. 

Have you checked that the instance is not actually paused or scaling?

Are you able to connect with other client tools?

**Reply by SenthilkumarE** [reply]
I am using the latest version of Dax Studio . The same instance i can able to connect with SQL Server and other client tools.

**Reply by SenthilkumarE** [reply]
![image](https://user-images.githubusercontent.com/90666336/133232848-bfffc25a-225f-4d01-a7b6-45d8a807e45f.png)
